src/Simplic.FileStructure.Workflow.UI/WorkflowApplicationHelper.cs

```diff
@@ -170,6 +207,11 @@ public static GridInvokeMethodResult ForwardCopyTo(GridFunctionParameter paramet
                     WorkflowId = workflowId,
                     Guid = Guid.NewGuid()
                 };
+                if ((string)itemBox.GetSelectedItemCell("Type") == "WorkflowOrganizationUnit")
```

Comment:
I would separate between some kind of internal-type and diplay-type. And better use ToString than (string)